### ISSUE TITLE
Dataparralel Issue

### ISSUE THREAD

**Issue by lqzhao** [xinge008/Cylinder3D#5]
Hi, it's a great work! 
Since the batch size can only be set to 1 when running the code on a 1080Ti GPU, I'm trying to use the data parralel trick to increase the number of training batch_size. However, I found when I set the batchsize to 2, the outputs of the model became a tensor of the shape ([4,20,480,360,32]) while the label is still ([2,480,360,32]). This conflict would rise an error of size mismatching.
Have you meet this problem before? Why would this happen?

**Comment by xinge008** [comment]
In fact, I haven't try the dataparallel due to the limited GPUs. It is better to show some code snippets for a detailed description.

**Comment by lqzhao** [comment]
```
            # forward + backward + optimize
            outputs = my_model(train_pt_fea_ten, train_vox_ten, train_batch_size)
            loss = lovasz_softmax(torch.nn.functional.softmax(outputs), point_label_tensor, ignore=0) + loss_func(
                outputs, point_label_tensor)
```
When I set the batchsize to 2, **outputs ** tensor has  a shape of ([4,20,480,360,32]), while **point_label_tensor** is a tensor of shape ([2,480,360,32]).

**Comment by rancheng** [comment]
you can try Distributed Data Parallel, Data Parallel has a lot of bugs...

**Comment by cardwing** [comment]
The dataparallel error may be caused by the spconv. Please refer to [this issue](https://github.com/traveller59/spconv/issues/35) for more details.

**Comment by xinge008** [comment]
Please reopen the issue when you have the further try.
